pkg/json/encode.go

```diff
@@ -40,3 +40,27 @@ func (s Set) MarshalJSON() ([]byte, error) {
 	buf.WriteString(`]]`)
 	return buf.Bytes(), nil
 }
+
+func (cmr *CondMonitorParameters) UnmarshalJSON(p []byte) error {
+	var tmp []json.RawMessage
+	if err := json.Unmarshal(p, &tmp); err != nil {
+		return err
+	}
+	if err := json.Unmarshal(tmp[0], &cmr.DatabaseName); err != nil {
+		return err
```

Comment:
you may want to add the fact that you failed when searching attempting to unmarshal databaseName:

```
return fmt.Errorf("Unmarshal database_name: %s", err)
```

Reply:
Done